pkg/policy/api/rule_validation.go

```diff
@@ -81,10 +81,22 @@ func (i *IngressRule) sanitize() error {
 }
 
 func (e *EgressRule) sanitize() error {
+
+	// TODO: GH-XXXX (support combining of label-based L3 and CIDR-based L3)
+	if len(e.ToCIDR) > 0 && len(e.ToEndpoints) > 0 {
+		return fmt.Errorf("Combining ToCIDR and ToEndpoints is not supported yet")
```

Comment:
I can remove this in favor of #3015?

Reply:
Yeah, mostly. I think there'll need to be a small followup to #3015 which rejects combinations of `ToEndpoints` with other L3 destinations, since `ToEndpoints` isn't available in master quite yet.